Specification.md

```diff
@@ -40,9 +40,11 @@ seq=11                                  \
 
 ## 2. Format and Naming
 
-**Sync Interest Format**: `/<group-prefix>/<state-vector>/<signature>`
+**Sync Interest Format**: `/<group-prefix>/v=2/<signature>`
 
-A state vector is appended to the name in TLV format:
+The State Vector is encoded in TLV format and included in the Application Parameters of the Sync Interest. The State Vector SHOULD be the first TLV block in the Application Parameters.
```

Comment:
Under TLV evolvability guidelines, it's not good to enforce a particular TLV being the first one, without a strong technical reason.
It's better to accept any ordering, with a recommendation that StateVector appearing before MappingData.

Reply:
> with a recommendation that StateVector appearing before MappingData

why the recommendation? unless a specific order is mandatory, a receiver cannot make any assumptions anyway, so why bother with a recommended order in the first place?

Reply:
The recommended ordering improves Interest aggregation. It is not enforced.

Reply:
I'm not sure interest aggregation is a concern here? it's a signed interest anyway...

Reply:
Signing with HMAC key and skipping all random fields in SigInfo => aggregation possible.

Reply:
We're not debating whether it's possible or not, but whether it's a concern. I'm saying "why bother". Moreover, this may even be application dependent, and the order itself (even if just a recommendation) doesn't matter as long as it's consistent.

Reply:
Just to be clear: I'm not against suggesting an order, I just don't think it's very useful or makes a meaningful difference in practice.

Reply:
I don't care honestly. But it's easier / better to recommend a order than recommending consistency for interoperability between vendors.